src/components/FromSTIX/Statistics.jsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import styles from "./from_stix.module.scss";
+import {useDispatch, useSelector} from "react-redux";
+import {clearMappings} from "../../store/actions/from_stix";
+import "./Statistics.scss";
+
+
+const Statistics = () => {
+    const dispatch = useDispatch();
+    const mapping = useSelector((state) => state.fromStix.mapping);
+
+
+    function calculateNumber(mapping){
+        var mylist = [];
+        for (let i = 0; i < Object.keys(mapping).length; i++) {
+            mylist.push((Object.keys(mapping))[i].split(':')[0]);
+        }
+        const unique = [...new Set(mylist)];
+        return unique.length - 0;
```

Comment:
why do we need to subtract zero here?

Reply:
I wrote in Line 21, instead of the zero it will be the number of custom object added. 
I will change the Zero after i will do the task of the custom object, 
I put a ZERO for now just to know where i will write the number of custom object.